ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-driving web app framework",
+  "homepage": "https://www.celljs.org",
+  "keywords": [],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/intercellular/cell.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "cell.js",
+          "cell.min.js"
+        ]
```

Comment:
@devsnek Thank you for your explanation.
We like to use `"cell*"` because it will more flexible when auto-update the new version release.
If someday release `cell.css`, then we don't revise this auto-update config, but it still works.
I think your suggestion is good, but what we just need is let this library host on CDNJS, isn't it? :)
I think we could let PeterBot work hard on this. :)